x-pack/legacy/plugins/ml/server/new_platform/plugin.ts

```diff
@@ -232,6 +221,19 @@ export class Plugin {
 
     initMlServerLog(logInitializationDeps);
     makeMlUsageCollector(plugins.usageCollection, coreSavedObjects);
+
+    plugins.licensing.license$.subscribe(async (license: any) => {
+      const { isEnabled: securityIsEnabled } = license.getFeature('security');
+      const { isAvailable, isEnabled } = license.getFeature(pluginId);
```

Comment:
this `isAvailable` check for the `ml` plugin returns false for a basic license, whereas ML should be available on basic with very reduced functionality (only the file data visualizer is currently available)
The original license check make this distinction.
As we're working towards server cutover, with the plan to completely review our licence checks, i'd suggest hard-coding `isAvailable` to `true` and have licensing temporarily broken until cutover is complete.
Seeing as we're relying of `XPackInfo` for additional license checking to control which features we enable in the UI, which will also be broken during the cutover, we will need to fix licensing as soon as the server has beeb cut over.